runtime/compiler/control/J9Options.cpp

```diff
@@ -1121,6 +1122,7 @@ static bool JITServerParseCommonOptions(J9JavaVM *vm, TR::CompilationInfo *compI
    int32_t xxJITServerSSLRootCertsArgIndex = FIND_ARG_IN_VMARGS(STARTSWITH_MATCH, xxJITServerSSLRootCertsOption, 0);
    int32_t xxJITServerUseAOTCacheArgIndex = FIND_ARG_IN_VMARGS(EXACT_MATCH, xxJITServerUseAOTCacheOption, 0);
    int32_t xxDisableJITServerUseAOTCacheArgIndex = FIND_ARG_IN_VMARGS(EXACT_MATCH, xxDisableJITServerUseAOTCacheOption, 0);
+   int32_t xxJITServerRequireServerArgIndex = FIND_ARG_IN_VMARGS(EXACT_MATCH, xxJITServerRequireServerOption, 0);
```

Comment:
Ideally you need to search for `-XX:-JITServerRequireServer` and see which of the two options appears last.
BTW, doesn't -XX:+RequireJITServer` sound better?

Reply:
It does sound better, but all other jitserver options start with `JITServer*` so I just used the same naming. I'll change it if that's good with you.

Reply:
Let's changes because we already have `UseJITServer` which breaks the pattern.